### PR TITLE
Styled components

### PR DESCRIPTION
**Pull request by amvaid**
@zcoursey22 Could you review this please. I just recreated the navbar with styled-components

### CODE REVIEW

**Comment by amvaid** [comment]
@despinoz Can you review this?

File: client/components/navbar.jsx [hunk 1]
```diff
@@ -1,30 +1,115 @@
...
+const Horizontal = styled.hr`
+  border-color: #034752;
+`;
 
-    <hr />
-  </div>
-);
 
 module.exports = NavBar;
+
```
Comment (despinoz) [inline client/components/navbar.jsx]: It seems that you have the same code above.

**Review by despinoz — COMMENTED** [review]
@amvaid I left you a note on a commented code. Also can you explain me why do you have a scss file if working with styled components.

File: client/components/navbar.jsx (hunk 1 above)
Comment (amvaid) [reply client/components/navbar.jsx]: @despinoz I made some changes that I thought would be better closer to the actual navbar on kickstarter.

**Comment by amvaid** [comment]
@despinoz I responded to your comment. I've made changes to better suit the goal of component. I'm leaving the scss file for now because I have a few other items that are using it. I'm going to delete the file once I've moved over to styled-components completely.

**Review by despinoz — APPROVED** [review]
Looks good

**Pull request merged**